### PR TITLE
release: v1.8.0

### PR DESCRIPTION
**Pull request by raelga**
Upgrades the operator with the latest helm chart available, [cert-manager@1.8.0](https://artifacthub.io/packages/helm/cert-manager/cert-manager/1.8.0).

Screenshot 2022-05-31 at 12 35 02
Screenshot 2022-05-31 at 12 44 28
Screenshot 2022-05-31 at 12 44 50

/kind release
/priority important-soon
/assign

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -29,12 +29,16 @@ make container-push
...
+A real example of an alpha release can be found at [release: v1.8.0-alpha.1](https://github.com/3scale-ops/cert-manager-helm-operator/commit/e3ac092d7c07fd3ca631dcd2418ea578a3c10221) commit.
+
 ## Stable
 
 * Execute the following target to create appropiate `alpha` and `stable` bundle files:
 
 ```bash
 make prepare-stable-release
 ```
+A real example of an alpha release can be found at [release: v1.8.0-alpha.1](https://github.com/3scale-ops/cert-manager-helm-operator/commit/e3ac092d7c07fd3ca631dcd2418ea578a3c10221) commit.
```
Comment (slopezz) [inline docs/release.md]: I guess here you wanted to add a commit for stable instead of copy/paste previous alpha??

Comment (raelga) [reply docs/release.md]: Yep, was accidentally left outside the commit.

**Pull request merged**